lib/model/store.dart

```diff
@@ -311,6 +313,8 @@ class PerAccountStore extends ChangeNotifier with ChannelStore, MessageStore {
   final Map<String, RealmDefaultExternalAccount> realmDefaultExternalAccounts;
   Map<String, RealmEmojiItem> realmEmoji;
   List<CustomProfileField> customProfileFields;
+  /// For docs, please see [InitialSnapshot.emailAddressVisibility].
+  final EmailAddressVisibility? emailAddressVisibility; // TODO(server-7): remove
```

Comment:
This isn't quite right because in this revision it doesn't get updated when the setting changes. This value is an example of a realm setting:
* https://zulip.com/api/get-events#realm-update_dict
* #668 

But I don't want dealing with that to block this feature, or indeed to block launching the app. It's rare this setting will change, and the consequences of having an out-of-date value for a while are small. So let's just mark it with a TODO comment:
```suggestion
  final EmailAddressVisibility? emailAddressVisibility; // TODO(#668) update this realm setting
```

(The TODO-server comment can be omitted because it's effectively covered by the one on InitialSnapshot.emailAddressVisibility — when we go to remove that, we'll necessarily notice we need to remove this at the same time.)